setup.py

```diff
@@ -1,8 +1,9 @@
-import setuptools
-from setuptools import find_packages
 import re
 from pathlib import Path
 
+import setuptools
```

Comment:
Just to let you know. We are actually dropping `setup.py` file with `supervision-0.12.0` release. So you should not depend on any changes made in this file.

Reply:
Yeah sounds great! Thank you :)